Scripts/Engine/Timer.lua

```diff
@@ -120,12 +120,12 @@ Timer = {
 				t.remainingTime = t.remainingTime - dt
 
 				if t.remainingTime <= 0 then
-					t.func(table.unpack(t.funcArgs))
 					if not t.loop then
 						t.active = false
 					else
 						t.remainingTime = t.remainingTime + t.totalTime
 					end
+					t.func(table.unpack(t.funcArgs))
```

Comment:
Why order was changed here? I am not saying it's a bug, just being curious :)

Reply:
It was a fix for https://github.com/MontyTRC89/TombEngine/issues/940; I forgot to switch to master before committing